units/src/amount.rs

```diff
@@ -1835,6 +1852,48 @@ mod tests {
     #[test]
     fn from_int_btc_panic() { Amount::from_int_btc(u64::MAX); }
 
+    #[test]
+    fn test_signed_amount_try_from_amount() {
+        let ua_positive = Amount::from_sat(123);
+        let sa_positive = SignedAmount::try_from(ua_positive).unwrap();
+        assert_eq!(sa_positive, SignedAmount(123));
+
+        let ua_zero = Amount::ZERO;
+        let sa_zero = SignedAmount::try_from(ua_zero).unwrap();
+        assert_eq!(sa_zero, SignedAmount(0));
+
+        let ua_max = Amount::MAX;
+        let sa_max = SignedAmount::try_from(ua_max);
+        assert_eq!(
+            sa_max,
+            Err(ParseAmountError::OutOfRange(OutOfRangeError {
+                is_signed: true,
+                is_greater_than_max: true
+            }))
+        );
+    }
+
+    #[test]
+    fn test_amount_try_from_signed_amount() {
+        let sa_positive = SignedAmount(123);
+        let ua_positive = Amount::try_from(sa_positive).unwrap();
+        assert_eq!(ua_positive, Amount::from_sat(123));
+
+        let sa_zero = SignedAmount(0);
+        let ua_zero = Amount::try_from(sa_zero).unwrap();
+        assert_eq!(ua_zero, Amount::ZERO);
+
+        let sa_negative = SignedAmount(-123);
+        let ua_negative = Amount::try_from(sa_negative);
+        assert_eq!(
+            ua_negative,
+            Err(ParseAmountError::OutOfRange(OutOfRangeError {
+                is_signed: false,
+                is_greater_than_max: false
+            }))
+        );
+    }
```

Comment:
I don't think tests are important for such trivial code but I don't object to having them.